packages/sanity/src/_internal/cli/commands/backup/listBackupCommand.ts

```diff
@@ -0,0 +1,137 @@
+import type {CliCommandDefinition} from '@sanity/cli'
+import {Table} from 'console-table-printer'
+import {isAfter, isValid, lightFormat, parse} from 'date-fns'
+import resolveApiClient from '../../actions/backup/resolveApiClient'
+import {defaultApiVersion, validateLimit} from './backupGroup'
+
+const DEFAULT_LIST_BACKUP_LIMIT = 30
+
+interface ListDatasetBackupFlags {
+  before?: string
+  after?: string
+  limit?: string
+}
+
+type ListBackupRequestQueryParams = {
+  before?: string
+  after?: string
+  limit: string
+}
+
+type ListBackupResponse = {
+  backups: ListBackupResponseItem[]
+}
+
+type ListBackupResponseItem = {
+  id: string
+  createdAt: string
+}
+
+const helpText = `
+Options
+  --limit <int>     Maximum number of backups returned. Default 30.
+  --after <string>  Only return backups after this date (inclusive)
+  --before <string> Only return backups before this date (exclusive). Cannot be younger than <after> if specified.
+
+Examples
+  sanity backup list DATASET_NAME
+  sanity backup list DATASET_NAME --limit 50
+  sanity backup list DATASET_NAME --after 2024-01-31 --limit 10
+  sanity backup list DATASET_NAME --after 2024-01-31 --before 2024-01-10
+`
+
+const listDatasetBackupCommand: CliCommandDefinition<ListDatasetBackupFlags> = {
+  name: 'list',
+  group: 'backup',
+  signature: '[DATASET_NAME]',
+  description: 'List available backups for a dataset.',
+  helpText,
+  action: async (args, context) => {
+    const {output, chalk} = context
+    const flags = args.extOptions
```

Comment:
same comment here, it would be good to re-parse the arguments using `yargs`

Reply:
This is fixed as well.